cypress/e2e/embedded_superset_dashboard.cy.js

```diff
@@ -0,0 +1,210 @@
+                })
+            } else if (req.url.includes(`${SUPERSET_BASE_URL}/embedded/`)) {
+                req.reply(SUPERSET_DASHBOARD_STUB)
+            } else {
+                // Just return the response by default
+                req.continue((resp) => resp)
+            }
+        })
+    })
+
+    it('creates an embedded superset dashboard', () => {
+        // Start a new dashboard from the start page
+        cy.visit('#/start')
+
+        cy.get(newButtonSel, EXTENDED_TIMEOUT).click()
+
+        // Choose the embedded dashboard option
+        cy.contains('External: Data from another source')
+            .should('be.visible')
+            .click()
+
+        // Click the configure source button
+        cy.contains('Configure source').should('be.visible').click()
+
+        // A modal form to create a new embedded dashboard is showing
+        cy.contains(
+            'New dashboard: configure external source (superset)'
+        ).should('be.visible')
+
+        // Check all initial values and change them
+        getInputByLabelText('Title').should('have.value', '').type(NAME)
+        getInputByLabelText('Code').should('have.value', '').type(CODE)
+        getInputByLabelText('Description', 'textarea')
+            .should('have.value', '')
+            .type(DESCRIPTION)
+        getInputByLabelText('Superset Embed ID')
+            .should('have.value', '')
+            .type(UUID)
+        getInputByLabelText('Show chart controls on dashboard items')
+            .should('be.checked')
+            .uncheck()
+        getInputByLabelText('Expand filters').should('not.be.checked').check()
+
+        // Click the create button
+        cy.contains('Save dashboard').should('be.enabled').click()
+
+        cy.contains('h3', NAME).should('be.visible')
+        cy.contains('External data').should('be.visible')
+        // An iframe should be visible with the UUID in the src
+        cy.get('iframe')
+            .should('be.visible')
+            .and('have.attr', 'src')
+            .and('contain', UUID)
+    })
+
+    it('has some options disabled in the action-bar', () => {
+        // Primary actions
+        cy.contains('button', 'Slideshow').should('be.disabled')
+        cy.contains('button', 'Filter').should('be.disabled')
+        // Actions in the more-menu
+        cy.getByDataTest('more-actions-button').should('be.enabled').click()
+        cy.contains('a', 'Make available offline').should(
+            'have.attr',
+            'aria-disabled',
+            'true'
+        )
+        cy.contains('a', 'Print')
+            .should('have.attr', 'aria-disabled', 'true')
+            .and('have.attr', 'aria-expanded', 'false')
+        // Close the menu by clicking the backdrop
+        cy.get('.backdrop').should('be.visible').click()
+    })
+
+    it('shows the description', () => {
```

Comment:
I'm a little concerned about changing the show description and letting it linger. I get what you're doing, but we also have another test suite for testing show/hide description. If these test suites run concurrently, they could interfere with each other.

Reply:
Good point, I hadn't considered this.